surveys/urls.py

```diff
@@ -6,53 +6,51 @@
 
 """
 from django.conf.urls import url, include
+from django.urls import path
 from django.utils.translation import ugettext_lazy as _
 from django.contrib.auth.decorators import login_required
-from surveys.views import AgencyCreate, LocationCreate, StudyCreate, \
-                            StudyAreaCreate, SurveyCreate, SurveyPublish, \
-                            SurveyListEdit, SurveyListRun, Signup, SurveySubmittedList, \
-                            SurveySubmittedDetail
+from surveys import views
```

Comment:
Rather than add another import to this long list, I did a quick refactor to use the module name to reference the view methods.

Reply:
Thanks! I'd been thinking about doing this.